### ISSUE TITLE
"Raycaster.camera" needs to be set in order to raycast against sprites.

### ISSUE THREAD

**Issue by kylekanouse** [vasturiano/3d-force-graph-vr#21]
**Describe the bug**
Web example using images for nodes is broken and does not render properly

console error message:

`Raycaster.camera" needs to be set in order to raycast against sprites.`

aslo 

```
Uncaught TypeError: Cannot read property 'matrixWorld' of null
    at kl.raycast (3d-force-graph-vr:2)
    at up (3d-force-graph-vr:2)
    at up (3d-force-graph-vr:2)
    at lp.intersectObjects (3d-force-graph-vr:2)
    at n.checkIntersections (3d-force-graph-vr:83)
    at n.tock (3d-force-graph-vr:83)
    at HTMLElement.value (3d-force-graph-vr:83)
    at t.object3D.onAfterRender (3d-force-graph-vr:83)
    at dl.render (3d-force-graph-vr:2)
    at HTMLElement.value (3d-force-graph-vr:83)
```

**To Reproduce**

1. Go to [https://vasturiano.github.io/3d-force-graph-vr/example/img-nodes/](https://vasturiano.github.io/3d-force-graph-vr/example/img-nodes/)

I am also able to reproduce in a local environment

**Expected behavior**
Should render graph with image icons for nodes as exampled [here]( https://vasturiano.github.io/3d-force-graph/example/img-nodes/)

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser [brave, safari]
 - Version [
     brave: Version 1.25.68 Chromium: 91.0.4472.77 (Official Build) (x86_64),
     safari: Version 14.0.3 (15610.4.3.1.7, 15610)
]

Thank you as I really appreciate this project.

**Comment by DubiousTunic** [comment]
Any hope of getting this fixed? I found that using an older version which i had magickally saved works. I love your work btw!

**Comment by antonkulaga** [comment]
Looks like it is the same bug as I reported https://github.com/vasturiano/three-spritetext/issues/31
@vasturiano can you take a look at this? It brokes a lot of projects :(

**Comment by antonkulaga** [comment]
I checked when regression appeared, the last version that works properly is 1.35.6

**Comment by vasturiano** [comment]
The cause of the issue is that Aframe has trouble raycasting against Sprite type objects. 

I've made a patch to this module that disables raycasting unless any of the interaction props are set (node/link click/hover methods).
This will allow you to get around this issue, by not setting any click/hover interaction when using sprites for nodes.

Please use the latest published version: `2.0.7`.
